spec/codelog/command/new_spec.rb

```diff
@@ -2,53 +2,51 @@
 
 describe Codelog::Command::New do
   describe '#run' do
+    let(:name) { 'change' }
+    let(:options) { {} }
+
+    subject { described_class.new name, options }
+    
     before :each do
       allow(Time).to receive_message_chain(:now, :strftime) { '20180119134323984' }
       allow(subject).to receive(:puts)
       allow(subject).to receive(:system) { true }
-      subject.run(options)
+      subject.run
     end
 
-    context 'with no additional options' do
```

Comment:
I think it makes sense to preserve this context. I think that mixing examples and contexts at the same nesting level is kinda awkward.

Reply:
I think I get what you tried to do: 
Leaving these examples at the root level might signal that they should apply independently of the context. However this is not what happens; as these examples are not re-run in each context, they effectively are only run for the "default context" and any regression made by a different context will not be caught

Reply:
I believe that replicating the common examples in each context is better than DRYing them up and running expectations inside a `before` block

Reply:
So, in your opinion, is it better to have another context mixing all the options(the `edit` and the custom name)?

Reply:
I think *mixing* all the `options` X `name` is not necessary. One context for each possible `options` or `name` would be enough